client/lib/domains/utils/parse-domain-against-tld-list.js

```diff
@@ -12,8 +7,9 @@ export function parseDomainAgainstTldList( domainFragment, tldList ) {
 		return domainFragment;
 	}
 
-	const parts = split( domainFragment, '.' );
-	const suffix = join( drop( parts ), '.' );
+	const parts = domainFragment.split( '.' );
+	parts.shift();
+	const suffix = parts.join( '.' );
```

Comment:
It's also possible to do `parts.slice( 1 ).join( '.' )`, where the methods can be chained 😎 `shift()` continues to be fine, just pointing out an alternative.

Reply:
Good point. I don't have a preference, and the `slice` looks a bit simpler, so I've used that instead. Thanks 😉